app/middleware/getPharmacies.js

```diff
@@ -3,6 +3,14 @@ const log = require('../lib/logger');
 const dedupe = require('../lib/dedupePharmacies');
 const constants = require('../lib/constants');
 
+function logZeroResults(res) {
```

Comment:
Is this the best place to do the logging or do it as its' own middleware? I'm struggling to choose 🤔 
What do you reckon?